### ISSUE TITLE
java.lang.NullPointerException on shutdown

### ISSUE THREAD

**Issue by Moosheplusplus** [powerbot/powerbot#1337]
```
[SEVERE] java.lang.NullPointerException 
	org.powerbot.script.rt6.Component.else(xd:512)
	org.powerbot.script.rt6.Component.textureId(xd:95)
	org.powerbot.script.rt6.Hud.this(vc:284)
	org.powerbot.script.rt6.Hud.else(vc:6)
	org.powerbot.script.rt6.Hud.else(vc:86)
	org.powerbot.script.rt6.Hud$1.run(vc:201)
```

occurs only when I am logged in and use File -> Exit as far as I can tell.

**Comment by Timer** [comment]
```
[SEVERE] java.lang.NullPointerException 
    org.powerbot.script.rt6.Component.isInScrollableArea(Component.java:471)
    org.powerbot.script.rt6.Component.textureId(Component.java:104)
    org.powerbot.script.rt6.Hud.opened(Hud.java:314)
    org.powerbot.script.rt6.Hud.updateBounds(Hud.java:38)
    org.powerbot.script.rt6.Hud.updateBounds(Hud.java:13)
    org.powerbot.script.rt6.Hud$1.run(Hud.java:64)
```

**Comment by Timer** [comment]
fixed in 8327e0d1395e511bb195c068e7e12a90ed4fd2fd
